### ISSUE TITLE
Go get fails when getting golang.org/x/tools/go/types

### ISSUE THREAD

**Issue by dfuentes** [golang/lint#96]
Go get is currently giving me errors related to golang.org/x/tools:

```bash
$ go get github.com/golang/lint/golint
# golang.org/x/tools/go/types
../../../golang.org/x/tools/go/types/initorder.go:140: syntax error: unexpected range, expecting {
../../../golang.org/x/tools/go/types/initorder.go:148: non-declaration statement outside function body
../../../golang.org/x/tools/go/types/initorder.go:149: syntax error: unexpected }
```

**Comment by rgarcia** [comment]
It looks like a recent commit started using a go 1.4 syntax feature https://github.com/golang/tools/commit/44ee65f5451d813b9795358b974e5798777f9e21#diff-c6c87a65a521b8a45c0bb52d2c358809R140

**Comment by dsymonds** [comment]
I've asked for that particular change to be rolled back (https://go-review.googlesource.com/2390). Let's see how that goes. You'll need to update to Go 1.4 eventually, but hopefully not today.

**Comment by dsymonds** [comment]
Run `go get -u golang.org/x/tools/go/types` now and you should pick up golang/tools@4fbaee5, which makes that package compile with Go 1.3 again.

**Comment by dfuentes** [comment]
Thanks
